app/views/generic_files/_generic_file.html.erb

```diff
@@ -1,6 +1,7 @@
 <li>
   <h2>
     <%= link_to generic_file.title_or_label, sufia.generic_file_path(generic_file) %>
+    <small>[<%= link_to 'Download', download_path(generic_file), target: '_blank', rel: 'noopener noreferrer' %>]</small>
```

Comment:
Ok, I've found a couple of other dropdown menus that will need this change as well. I'll fix those and let you know when I've pushed a new version